pkg/operator/argument_test.go

```diff
@@ -43,15 +44,11 @@ func TestBuildArgs(t *testing.T) {
 
 	for _, arg := range args {
 		argString := fmt.Sprintf("--%s=%s", arg.Name, arg.Value)
-		if !slices.Contains(containerArgs, argString) {
-			t.Fatalf("expected containerArgs to contain arg %v, got %v", argString, containerArgs)
-		}
+		require.True(t, slices.Contains(containerArgs, argString), "expected containerArgs to contain arg %v, got %v", argString, containerArgs)
```

Comment:
Could we use [require.Contains](https://pkg.go.dev/github.com/stretchr/testify/require#Contains) instead?

Reply:
Yeah, I just thought maybe True and False would be more intuitive. Let me fix that